src/vcpkg/commands.ci.cpp

```diff
@@ -400,6 +430,18 @@ namespace vcpkg
         LocalizedString not_supported_regressions;
         {
             std::string msg;
+            
+            // First, print ports that were excluded early (before dependency resolution)
+            for (const auto& excluded_spec : ports_to_exclude)
```

Comment:
This is a good idea I missed in my version of the same https://github.com/microsoft/vcpkg-tool/pull/1822#issuecomment-3429660695